.github/workflows/check-link-rot.yaml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+
+name: check-link-rot
+
+jobs:
+  check-link-rot:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      R_KEEP_PKG_SOURCE: yes
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-pandoc@v2
```

Comment:
Is pandoc necessary here?

Reply:
Yes, definitely. I had tried without (see commit history in this [experimental PR](https://github.com/IndrajeetPatil/minimalReprex/pull/8)) and it fails.